src/test/kotlin/uk/gov/justice/digital/hmpps/crimeportalgateway/integration/endpoint/ExternalDocRequestEndpointIntTest.kt

```diff
@@ -70,16 +72,62 @@ class ExternalDocRequestEndpointIntTest : IntegrationTestBase() {
         verify(sqsService).enqueueMessage(anyString())
     }
 
-    @Disabled
     @Test
-    fun `given invalid ExternalDocumentRequest should provide SOAP fault `() {
-        val requestEnvelope: Source = StringSource(externalDocRequest.replace("<documents></documents>", "xxx"))
+    fun `should not enqueue message when court is not processed but return acknowledgement`() {
+        val externalDoc1 = readFile("src/test/resources/soap/ignored-courts.xml")
+        val requestEnvelope: Source = StringSource(externalDoc1)
 
         mockClient.sendRequest(RequestCreators.withSoapEnvelope(requestEnvelope))
-            .andExpect(serverOrReceiverFault())
+            .andExpect(validPayload(xsdResource))
+            .andExpect(
+                xpath("//ns3:Acknowledgement/ackType/MessageComment", namespaces)
+                    .evaluatesTo("Message ignored - the court B10XX in the message is not processed")
```

Comment:
Do we know that CGI will ignore the message returned in the Ack? One would imagine so and definitely nice to be explicit about what's happening but might be worth checking if we're not sure.